src/Drivers/DatabaseDriver.php

```diff
@@ -276,24 +313,13 @@ public function setForAllScopes($feature, $value): void
      */
     protected function update($feature, $scope, $value)
     {
-        $exists = $this->newQuery()
-            ->where('name', $feature)
-            ->where('scope', $serialized = Feature::serializeScope($scope))
-            ->exists();
-
-        if (! $exists) {
-            return false;
-        }
-
-        $this->newQuery()
+        return (bool) $this->newQuery()
             ->where('name', $feature)
-            ->where('scope', $serialized)
+            ->where('scope', Feature::serializeScope($scope))
             ->update([
                 'value' => json_encode($value, flags: JSON_THROW_ON_ERROR),
                 static::UPDATED_AT => Carbon::now(),
             ]);
-
-        return true;
```

Comment:
This was needlessly doing two queries.

Updated to only do a single update statement.